08_files_lab/01.py

```diff
@@ -1,6 +1,16 @@
-"""
-Write a program that takes 2 file names
-and appends the second file's contents to
-the end of the first one
-"""
-
+#Commited by Shlomit
+#D:\tmp\t1.txt D:\tmp\t2.txt
+
+import sys
+if len(sys.argv) < 3:
+    print "error -not enugh args"
+    sys.exit

```

Comment:
1. Line 7 does nothing. To actually call sys.exit you need to write parenthesis after function name such as `sys.exit()`

2. Better to write the error message inside `sys.exit()`. It prints the error to stderr AND returns an error code to the operating system.